codegen/src/main/java/software/amazon/awssdk/codegen/internal/Utils.java

```diff
@@ -70,9 +70,15 @@ public static boolean isExceptionShape(Shape shape) {
 
     public static boolean isOrContainsEnumShape(Shape shape, Map<String, Shape> allShapes) {
         boolean isEnum = isEnumShape(shape);
+        // FIXME: This test does not cover nesting of containers; see commented test below
```

Comment:
Not sure if I understand this, the commented below are not test code though

Reply:
Ah my fault for the confusing choice of words. I just meant test in the general meaning like "check"; e.g. check if this contains an enum